### PR TITLE
Triplelift native (DO NOT MERGE)

### PR DESCRIPTION
**Pull request by Kevin-P-Kerr**
request from `origin` to `prebid s2s`
```
{ "id": "some-request-id",
   "site": {
    "page": "https://www.123greetings.com/events/fathers_day/happy_fathers_day",
    "publisher": {
      "id": "pub-4627517680249670",
      "ext": {
        "country": "US"
      }
    } 
    },
    "imp": [
      {
        "tagid" : "makersalley_plain",
        "id": "some-impression-id",
   "native":{
    "request" : "{\"plcmtcnt\":1,\"plcmttype\":2,\"privacy\":1,\"context\":1,\"contextsubtype\":12,\"eventtrackers\":[{\"event\":1,\"methods\":[1,2]},{\"event\":2,\"methods\":[1]}],\"assets\":[{\"data\":{\"type\":12},\"required\":1},{\"title\":{\"len\":50},\"required\":1},{\"img\":{\"w\":80,\"h\":80,\"type\":1},\"required\":1},{\"img\":{\"w\":1200,\"h\":627,\"type\":3},\"required\":1},{\"data\":{\"type\":3},\"required\":0},{\"data\":{\"len\":100,\"type\":2},\"required\":1},{\"video\":{\"mimes\":[\"video/mpeg\",\"video/mp4\"],\"minduration\":2,\"protocols\":[2,5],\"maxduration\":2,\"ext\":{\"playbackmethod\":[1,2]}},\"required\":1}],\"ver\":\"1.2\"}"
   },

        "ext": {
          "triplelift" : { "inventoryCode":"seasonandsuppers_main_right"} 
        }
      }
    ],
    "tmax": 500
  }
````
response from us to prebid
```
{
  "id": "some-request-id",
  "seatbid": [
    {
      "bid": [
        {
          "id": "159429333415296977170",
          "impid": "some-impression-id",
          "price": 3.071,
          "nurl": "https://tlx.3lift.net/s2sn/notify?px=1&ts=1570645454&aid=159429333415296977170&ec=1540_73739_2763332&n=GqwBaHR0cDovL2V4Y2hhbmdlLXRlc3Rlci50cmlwbGVsaWZ0Lm5ldDo4MDc3L25vdGlmeT9pZD0xNTk0MjkzMzM0MTUyOTY5NzcxNzAmcHJpY2U9JHtBVUNUSU9OX1BSSUNFfSZhYmlkPTE1NjMwOTg1MzgyMDg2NzY2MDcmYWltcGlkPTE1NjMwOTg1MzgyMDg2NzY2MDctMS1udWxsJnR5cGU9QmlkMjJTZWF0c%2FICoQEIABIVMTU5NDI5MzMzNDE1Mjk2OTc3MTcwGAAgASiEDDCLwARAAUgAUABgAWgAcJijBZABAJgBAKgBs8RgsAH1JLgB6AfAAf8XyAH1JOAB%2FxfwAQD4AY4qgAKvG4gC%2FxeRAgAAAAAAAPA%2FmQJmZmZmZmbWP6ECAAAAAAAAAACoAgCwAgjIAgLYAgDxAmZmZmZmZuY%2F%2BAKEApADAJgDAKADAPgCFIgDAJIDBGVjZGQ%3D",
          "adm": "{\"native\":{\"ver\":1.2,\"assets\":[{\"id\":1,\"data\":{\"value\":\"Renaissance Curacao Seat 1\"}},{\"id\":5,\"data\":{\"value\":\"Discover the hidden gem of the Caribbean\"}},{\"id\":3,\"img\":{\"url\":\"https://img.3lift.net/?width=1200&height=627&url=%2F%2Fimages.3lift.net%2F1581619.jpg&v=17&logo=%2F%2Fimages.3lift.net%2F1564213.jpg&logo_x_pct=3&logo_y_pct=3\",\"w\":760,\"h\":375}},{\"id\":6,\"video\":{}}],\"link\":{\"url\":\"http://ag.yieldoptimizer22222.com/ag/event?type=click&gi=vHH2MVL_SjnGKP62Cib00jK0s2o-vYF4fY9KN-ZMFBizsiWz6ErQ6BZVg_BuhEdSMwuOZFakknl2X_IMznDSZwult3EAcf-2t5bLa92B_a55wqXvj8LVNcUw5THdrYVDkzGglcgDIGNJBHMXOMtXE8627ymU1e3DANpWp1TQam6TMsQptW1GkKjvN34QhFK0&ref_url=123greetings.com\"},\"eventtrackers\":[{\"event\":1,\"method\":1,\"url\":\"https://eb2.3lift.net/r?rr=creative&aid=159429333415296977170&impid=159429333415296977170_0\"},{\"event\":1,\"method\":1,\"url\":\"https://ad.doubleclick22222.net/ad/N4406.282142.ADARAMEDIAINC./B8015471.114430570;sz=1x1;ord=142248259412135895294\"}]}}",
          "adomain": [
            "marriott.com"
          ],
          "iurl": "//images.3lift.net/1581619.jpg",
          "crid": "1540_73739_2763332"
        }
      ]
    }
  ],
  "cur": "USD"
}
```
response from prebid to origin
```
{
  "id": "some-request-id",
  "seatbid": [
    {
      "bid": [
        {
          "id": "159429333415296977170",
          "impid": "some-impression-id",
          "price": 3.071,
          "nurl": "https://tlx.3lift.net/s2sn/notify?px=1&ts=1570645454&aid=159429333415296977170&ec=1540_73739_2763332&n=GqwBaHR0cDovL2V4Y2hhbmdlLXRlc3Rlci50cmlwbGVsaWZ0Lm5ldDo4MDc3L25vdGlmeT9pZD0xNTk0MjkzMzM0MTUyOTY5NzcxNzAmcHJpY2U9JHtBVUNUSU9OX1BSSUNFfSZhYmlkPTE1NjMwOTg1MzgyMDg2NzY2MDcmYWltcGlkPTE1NjMwOTg1MzgyMDg2NzY2MDctMS1udWxsJnR5cGU9QmlkMjJTZWF0c%2FICoQEIABIVMTU5NDI5MzMzNDE1Mjk2OTc3MTcwGAAgASiEDDCLwARAAUgAUABgAWgAcJijBZABAJgBAKgBs8RgsAH1JLgB6AfAAf8XyAH1JOAB%2FxfwAQD4AY4qgAKvG4gC%2FxeRAgAAAAAAAPA%2FmQJmZmZmZmbWP6ECAAAAAAAAAACoAgCwAgjIAgLYAgDxAmZmZmZmZuY%2F%2BAKEApADAJgDAKADAPgCFIgDAJIDBGVjZGQ%3D",
          "adm": "{\"native\":{\"ver\":1.2,\"assets\":[{\"id\":1,\"data\":{\"value\":\"Renaissance Curacao Seat 1\"}},{\"id\":5,\"data\":{\"value\":\"Discover the hidden gem of the Caribbean\"}},{\"id\":3,\"img\":{\"url\":\"https://img.3lift.net/?width=1200&height=627&url=%2F%2Fimages.3lift.net%2F1581619.jpg&v=17&logo=%2F%2Fimages.3lift.net%2F1564213.jpg&logo_x_pct=3&logo_y_pct=3\",\"w\":760,\"h\":375}},{\"id\":6,\"video\":{}}],\"link\":{\"url\":\"http://ag.yieldoptimizer22222.com/ag/event?type=click&gi=vHH2MVL_SjnGKP62Cib00jK0s2o-vYF4fY9KN-ZMFBizsiWz6ErQ6BZVg_BuhEdSMwuOZFakknl2X_IMznDSZwult3EAcf-2t5bLa92B_a55wqXvj8LVNcUw5THdrYVDkzGglcgDIGNJBHMXOMtXE8627ymU1e3DANpWp1TQam6TMsQptW1GkKjvN34QhFK0&ref_url=123greetings.com\"},\"eventtrackers\":[{\"event\":1,\"method\":1,\"url\":\"https://eb2.3lift.net/r?rr=creative&aid=159429333415296977170&impid=159429333415296977170_0\"},{\"event\":1,\"method\":1,\"url\":\"https://ad.doubleclick22222.net/ad/N4406.282142.ADARAMEDIAINC./B8015471.114430570;sz=1x1;ord=142248259412135895294\"}]}}",
          "adomain": [
            "marriott.com"
          ],
          "iurl": "//images.3lift.net/1581619.jpg",
          "crid": "1540_73739_2763332",
          "ext": {
            "prebid": {
              "type": "native"
            }
          }
        }
      ],
      "seat": "triplelift"
    }
  ],
  "ext": {
    "responsetimemillis": {
      "triplelift": 484
    },
    "tmaxrequest": 500
  }
}
```

### CODE REVIEW

**Comment by Kevin-P-Kerr** [comment]
this hits our `s2sn` endpoint with supply source id =20

**Comment by correa-brian** [comment]
so we're only supporting native over prebid s2s?

**Comment by Kevin-P-Kerr** [comment]
> so we're only supporting native over prebid s2s?

no, this is a branch that we are going to give msft. everyone else will use the mainline branch.

File: config/config.go [hunk 1]
```diff
@@ -641,7 +641,7 @@ func SetupViper(v *viper.Viper, filename string) {
 	v.SetDefault("adapters.rtbhouse.endpoint", "http://prebidserver-s2s-ams.creativecdn.com/bidder/prebidserver/bids")
 	v.SetDefault("adapters.somoaudience.endpoint", "http://publisher-east.mobileadtrading.com/rtb/bid")
 	v.SetDefault("adapters.sovrn.endpoint", "http://ap.lijit.com/rtb/bid?src=prebid_server")
-	v.SetDefault("adapters.triplelift.endpoint", "https://tlx.3lift.com/s2s/auction?supplier_id=19")
+	v.SetDefault("adapters.triplelift.endpoint", "https://tlx.3lift.com/s2sn/auction?supplier_id=20")
 	v.SetDefault("adapters.adkernel.endpoint", "http://{{.Host}}/hb?zone={{.ZoneID}}")
```
[the remark this answers is not in the document]
Comment (Kevin-P-Kerr) [reply config/config.go]: hey @guscarreon  sorry--this was an experimental branch.  please disregard!

[the remark this answers is not in the document]
Comment (Kevin-P-Kerr) [reply config/config.go]: though, @guscarreon  is it possible to route native requests to a different endpoint than the banner and video requests?

[the remark this answers is not in the document]
Comment (Kevin-P-Kerr) [reply config/config.go]: 👍  thanks for the info

**Pull request closed without merging**